mpd_client/src/commands/responses/sticker.rs

```diff
@@ -0,0 +1,91 @@
+use super::KeyValuePair;
+use std::collections::HashMap;
+
+/// Response to the [`sticker get`] command.
+///
+/// [`sticker get`]: crate::commands::definitions::StickerGet
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[non_exhaustive]
+pub struct StickerGet {
+    /// The sticker value
+    pub value: String,
+}
+
+impl StickerGet {
+    pub(crate) fn from_frame(raw: impl IntoIterator<Item = KeyValuePair>) -> Self {
+        let pair: String = raw.into_iter().map(|(_, value)| value).next().unwrap();
+
+        // server returns the key/value
+        // we know the key so just get the value
+        let value = pair.split_once('=').unwrap().1.to_string();
```

Comment:
This should return a `TypedResponseError` with the `InvalidValue` kind instead of panicking (same for the other impls). This could be extracted into a separate internal `parse_tag_value(String) -> Result<String>` function.